database/dict_converters/team_converter.py

```diff
@@ -3,7 +3,7 @@
 
 class TeamConverter(ConverterBase):
     SUBVERSIONS = {  # Increment every time a change to the dict is made
-        3: 4,
+        3: 5,
```

Comment:
@bdaroz Do we need to bump this?

Reply:
I don't believe so because the dict isn't changing -- we're still holding spot for the value -- just nulling it.

Also, the version number for the APIv3 Swagger spec (and changelog in it) should be bumped too.

Reply:
You do need to bump it.

Reply:
Can you point me in the right direction for those things?

Reply:
You already bumped it. Unless you're referring to Swagger in case @bdaroz chime in.

Reply:
Yep - talking about the Swagger version/changelog stuff

Reply:
Paging @bdaroz on this one

Reply:
At an event this weekend. Will reply after.